tests/testutils/test_output_line.py

```diff
@@ -19,7 +19,7 @@ def inner(confidence: Confidence = HIGH) -> Message:
         return Message(
             symbol="missing-docstring",
             msg_id="C0123",
-            location=[
+            location=[  # type: ignore
```

Comment:
This should probably be a `MessageLocationTuple`. Unfortunately, you'll still need `type: ignore` comments for `"line"` and `"column".

Reply:
Yes, but I want to change this with an upcoming refactor/typing of `OutputLine`. See the PR description. I can also do that first if you want me to, but I think it might be better to do this first and then do `OutputLine` and its tests.

Reply:
Sorry. Probably shouldn't review PRs that late 😅

Reply:
No worries! As you know it is truly appreciated!